libopenage/engine.cpp

```diff
@@ -548,4 +542,12 @@ void Engine::move_phys_camera(float x, float y, float amount) {
 	this->engine_coord_data->camgame_phys += cam_delta.to_phys3();
 }
 
+void Engine::start_game(std::unique_ptr<GameMain> game) {
+	// TODO: maybe implement a proper 1-to-1 connection
+	assert(game);
```

Comment:
Yes, the `error::Error` is fatal by design. If one inherits from that and uses it for code flow (which will happen someday for user input, mod packs, ...) then the backtrace collection can be disabled upon construction, this already works.

If parts of the code should stay exception neutral, then i'm ok with that part, but whenever a backtrace would make sense elsewhere, our `Error` is better.